astro.config.mjs

```diff
@@ -13,9 +14,19 @@ export default defineConfig({
 			dsn: process.env.SENTRY_DSN,
 			environment: process.env.NODE_ENV,
 			sampleRate: 1.0,
-			tracesSampleRate: 0.3,
-			replaysSessionSampleRate: 0.0,
-			replaysOnErrorSampleRate: 0.1,
+			tracesSampleRate: 0.5,
+			replaysSessionSampleRate: 0.05,
+			replaysOnErrorSampleRate: 0.5,
+			integrations: [
+				Sentry.replayIntegration({
```

Comment:
got .replayIntegration not a function during build
Maybe the `@sentry/astro` package is not the correct version?